src/modules/quizzes.js

```diff
@@ -88,7 +93,8 @@ class Quizzes {
    * @param {string} [parameters] - Additional parameters to refine result set
    * @param {string} [parameters.section] - Product catalog section
    * @param {array} [parameters.answers] - An array of answers in the format [[1,2],[1]]
-   * @param {string} [parameters.versionId] - Version identifier for the quiz
+   * @param {string} [parameters.quizVersionId] - Version identifier for the quiz
```

Comment:
What do you think about:
> Version identifier for the quiz. Version ID will be returned with the first request and it should be passed with subsequent requests. More information can be found: https://docs.constructor.io/rest_api/quiz/using_quizzes/#quiz-versioning

Reply:
I was thinking of doing that but I felt it was very verbose compared to the the rest of the comments we leave. But I will add it there if you feel like we need it.

Reply:
@jjl014 what do you think?

Reply:
I like being more verbose here at least for those parameters since there's some special logic behind it.

It seems like Stanley incorporated those changes already, so we're good there. 👍

Thank you guys!